<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_exams</class>
 <widget class="QDialog" name="edit_exams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QComboBox" name="responsible"/>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="Date">
       <property name="text">
        <string>Date :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="1">
        <widget class="QLineEdit" name="L_Year"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Month">
         <property name="text">
          <string>Month :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="L_Day"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Day">
         <property name="text">
          <string>Day :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" rowspan="2">
        <widget class="QLabel" name="Year">
         <property name="text">
          <string>Year :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2">
        <widget class="QLineEdit" name="L_Month"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>184</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="Combo_Module">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Chose Module :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QFormLayout" name="formLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_93">
       <property name="text">
        <string>Groups</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_30">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QCheckBox" name="g1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QCheckBox" name="g7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g10">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g11">
           <property name="text">
            <string>11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g12">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_79">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start :    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="start">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_86">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>to :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="end">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Time :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Responsible :</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCancelExam">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditExam">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="Auto">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
